### PR TITLE
Set Storybook as the landing page for Vocabulary

### PR DESCRIPTION
**Pull request by dhruvkb**
**Description**
This PR removes the landing website, sets Storybook as the landing website and makes the `npm` scripts similar to Vocabulary.

**Type of PR** 
This PR is a feature.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
@kgodey should I go ahead with this? This PR is the Vue Vocabulary correspondent of creativecommons/vocabulary#132.

File: .storybook/manager-head.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- Creative Commons favicon -->
+<link rel="icon" href="icons/favicon.png">
+
+<!-- Meta info for Open Graph -->
+<meta property="og:title" content="vue-vo·cab·u·lar·y">
+<meta property="og:type" content="website">
+<meta property="og:url" content="https://cc-vue-vocabulary.netlify.com/">
+<meta property="og:description" content="Vue components implementing a cohesive design system to unite the web facing Creative Commons">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/titlecard.png">
```
Comment (kgodey) [inline .storybook/manager-head.html]: This references Vocabulary, not Vue Vocabulary.

File: .storybook/manager-head.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<!-- Meta info for Twitter -->
+<meta name="twitter:title" content="vue-vo·cab·u·lar·y">
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:description" content="Vue components implementing a cohesive design system to unite the web facing Creative Commons">
+<meta name="twitter:image" content="https://cc-vocabulary.netlify.com/titlecard.png">
```
Comment (kgodey) [inline .storybook/manager-head.html]: This references Vocabulary, not Vue Vocabulary.

**Comment by kgodey** [comment]
Added a couple of comments but they don't need to block the PR.

**Pull request merged**